src/dances/radius.js

```diff
@@ -0,0 +1,9 @@
+export default (elem, value, options = {}) => {
+    const max = !isNaN(options.max) ? options.max : 25
+    const min = !isNaN(options.min) ? options.min : 0
+    let borderRadius = (max - min) * value
+    if(options.reverse) {
+      borderRadius = max - borderRadius
+    }
+    elem.style.borderRadius = `${borderRadius}px`
```

Comment:
Oh wait i just spotted the difference ! You forgot to add `+ min` here, if someone specify max = 50 and min = 25 it will have the same effect as max = 25 and min = 0

I guess reverse mode would be something like 
```js
if(options.reverse) {
    borderRadius = max - borderRadius
} else {
    borderRadius = min + borderRadius
}
```

Reply:
yes you are right. I did the change :)